cmd/beacon-chain/flags/base.go

```diff
@@ -263,4 +263,9 @@ var (
 		Usage: "Directory for the slasher database",
 		Value: cmd.DefaultDataDir(),
 	}
+	BlobRetentionEpoch = &cli.Uint64Flag{
+		Name:  "extend-blob-retention-epoch",
+		Usage: "Extend blob retention epoch period to beyond default 4096 epochs (~18 days). The node will error at start if input value is less than 4096 epochs.",
+		Value: uint64(params.BeaconNetworkConfig().MinEpochsForBlobsSidecarsRequest),
```

Comment:
Are params initialized at this stage of execution or will this just always show the mainnet default?

Reply:
Yeah, it's initialized. The default here is not used. I just set it anyway because it's weird if you use `--help`, then it says default is 0 other wise